### DISCUSSION TITLE
The first week

### DISCUSSION THREAD

**Discussion by randypacker** [dreamworksanimation/openmoonray#76] [Announcements]
Hello!

It’s been a great start to the MoonRay launch, and thanks everyone for your interest and engagement in this project.  It’s exciting to see a number of reports of the community successfully building and running MoonRay in the container and building in environments other than that of DreamWorks.  Given the larger scale of the repos, that’s an inspiring sign.  We also see reports of people still working to get everything up and running, but making progress nevertheless.

We are paying attention to all the issues and discussions, and are helping where we can.  We are definitely encouraged to see the community help each other out, which is a significant boost!  Note that internally we are a small team, supporting multiple productions on a schedule in our unique environment, and so don’t have as much opportunity to solve some diverse issues.  Put simply, we make a best effort, and hope any gaps can be filled with your help, and frankly, we’re excited to see in which directions the community takes it.

We also want to note that we’re still working through many of the fine details and strategies for our workflows around accepting contributions, external testing, releases, documentation versioning, and more.  We’re in this for the long term and hope to continue to learn, collaborate with you and improve all of the processes around MoonRay’s open source release for years to come.

We aimed high for our initial release, and even though these remaining details are important to solve soon, as people begin to explore MoonRay and its source and want to begin making contributions we also feel we had some time to plan these workflows carefully and therefore didn’t want to delay the initial release.

Thanks to everyone trying out MoonRay and stay tuned for future announcements!

**Comment by BrianHanke** [comment]
Congrats on a successful release! It's an exciting project and there's a ton of interest out here.

The build guide is excellent. It's relatively simple to get up and running with MoonRay standalone on CentOS 7.

I think a top priority should be creating a guide for getting MoonRay working in Houdini Solaris. The current docs give the impression that it should "just work," but based on my extensive tests it seems to be quite a difficult coding challenge. As far as I know, nobody in the community has succeeded. I got some very helpful feedback from SideFX on the topic, but of course they don't know the exact process either so I eventually hit a dead end.

**Comment by odil24** [comment]
@randypacker Hi. First of all congratulations! Is there have any test binary builds for Windows users for testing purposes?

**Reply by kjeffery** [reply]
I don't think anyone has yet built MoonRay for Windows. It's only been out for a week, and it is sure to be a fairly large project.

**Comment by samhodge** [comment]
Thanks for everything so far, its pretty well documented, sorry for not taking the time to read in depth before asking questions.

**Reply by kjeffery** [reply]
I don't read the documentation either, but I do have to write more of it.
